### PR TITLE
Initial unity project babyyyy

### PR DESCRIPTION
**Pull request by Nitrp**

### CODE REVIEW

**Comment by SprogisM** [comment]
what is this shit code
fix

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+﻿{
+    "files.exclude":
+    {
+        "**/.DS_Store":true,
+        "**/.git":true,
+        "**/.gitignore":true,
+        "**/.gitmodules":true,
+        "**/*.booproj":true,
+        "**/*.pidb":true,
+        "**/*.suo":true,
```
Comment (SprogisM) [inline .vscode/settings.json:10]: bruh

**Review by jakobemilandersson — APPROVED** [review]
Snyggt Gurra Mannen!

**Comment by SprogisM** [comment]
LGTM

File: .vscode/settings.json (hunk 1 above)
Comment (znauz) [reply .vscode/settings.json:10]: what is this code review??

**Pull request merged**